crates/libcgroups/src/stats.rs

```diff
@@ -317,76 +362,107 @@ pub fn parse_nested_keyed_data(file_path: &Path) -> Result<HashMap<String, Vec<S
     Ok(stats)
 }
 
-/// Parses a file that is structed according to the nested keyed format
-/// # Example
-/// ```
-/// use libcgroups::stats::parse_device_number;
```

Comment:
Do we need to keep the example here?

Reply:
The example was triggering doctets, which then failed because the function was made crate-only